### ISSUE TITLE
AttributeError: type object 'datetime.datetime' has no attribute 'datetime'

### ISSUE THREAD

**Issue by nadworny** [koxudaxi/datamodel-code-generator#298]
**Describe the bug**
Generating schemas with `format: date-time` leads to wrongly imported datetime module. While running a FastAPI app that uses generated model.py I'm getting the following error:
```
AttributeError: type object 'datetime.datetime' has no attribute 'datetime'
```

**To Reproduce**

Example schema:
```json

components:
  schemas:
    InventoryItem:
      required:
      - id
      - manufacturer
      - name
      - releaseDate
      type: object
      properties:
        id:
          type: string
          format: uuid
          example: d290f1ee-6c54-4b01-90e6-d701748f0851
        name:
          type: string
          example: Widget Adapter
        releaseDate:
          type: string
          format: date-time
          example: 2016-08-29T09:12:33.001Z
        manufacturer:
          $ref: '#/components/schemas/Manufacturer'
    Manufacturer:
      required:
      - name
      type: object
      properties:
        name:
          type: string
          example: ACME Corporation
        homePage:
          type: string
          format: url
          example: https://www.acme-corp.com
        phone:
          type: string
          example: 408-867-5309

```

Used commandline:
```
$ datamodel-codegen --input openapi.yaml --output model.py --input-file-type=openapi
```

**Expected behavior**
Imports should be resolved properly.
I fixed that by replacing the following lines:
```
from datetime import datetime
...
releaseDate: datetime = Field(
```

with these:
```
import datetime
...
releaseDate: datetime.datetime = Field(
```

**Version:**
 - OS: macos
 - Python version: 3.9.0
 - datamodel-code-generator version: 0.6.17

**Comment by koxudaxi** [comment]
@nadworny 
Thank you for creating this issue.
I found the bug.  The example of `releaseDate` was parsed to `datetime`. 
I expected the type is str.
But, The YAML parser cast the example to `datetime`. I fixed it.
I have released a new version as  `0.6.18`.


The result is here.
```python
# generated by datamodel-codegen:
#   filename:  openapi.yaml
#   timestamp: 2021-01-08T15:37:44+00:00

from __future__ import annotations

from datetime import datetime
from typing import Optional
from uuid import UUID

from pydantic import BaseModel, Field


class Manufacturer(BaseModel):
    name: str = Field(..., example='ACME Corporation')
    homePage: Optional[str] = Field(None, example='https://www.acme-corp.com')
    phone: Optional[str] = Field(None, example='408-867-5309')


class InventoryItem(BaseModel):
    id: UUID = Field(..., example='d290f1ee-6c54-4b01-90e6-d701748f0851')
    name: str = Field(..., example='Widget Adapter')
    releaseDate: datetime = Field(..., example='2016-08-29T09:12:33.001Z')
    manufacturer: Manufacturer

```

**Comment by nadworny** [comment]
Awesome, I'll test next week!
